dashboards/resource_lakeview_dashboard.go

```diff
@@ -0,0 +1,233 @@
+package dashboards
+
+import (
+	"bufio"
+	"context"
+	"crypto/md5"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"time"
+
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Reads the file content from a given path
+func readFileContent(source string) ([]byte, error) {
+	log.Printf("[INFO] Reading %s", source)
+	f, err := os.Open(source)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	reader := bufio.NewReader(f)
+	return io.ReadAll(reader)
+}
+
+// Calculates MD5 hash of the given content
+func calculateMd5Hash(content []byte) string {
+	return fmt.Sprintf("%x", md5.Sum(content))
+}
+
+// Reads content from a JSON string or a file path and returns the content and its MD5 hash
+func readSerializedJsonContent(jsonStr, filePath string) (serJSON string, md5Hash string, err error) {
+	var content []byte
+	if filePath != "" {
+		content, err = readFileContent(filePath)
+		if err != nil {
+			return "", "", err
+		}
+	} else {
+		log.Printf("[INFO] Reading `serialized_json` of %d bytes", len(jsonStr))
+		content = []byte(jsonStr)
+	}
+	md5Hash = calculateMd5Hash(content)
+	return string(content), md5Hash, nil
+}
+
+// ResourceLakeviewDashboard manages lakeview dashboards
+func ResourceLakeviewDashboard() common.Resource {
+	s := common.StructToSchema(dashboards.Dashboard{},
+		func(s map[string]*schema.Schema) map[string]*schema.Schema {
+			s["create_time"].Computed = true
+			s["dashboard_id"].Computed = true
+			s["display_name"].Optional = false
+			s["display_name"].Required = true
+			s["etag"].Computed = true
+			s["lifecycle_state"].Computed = true
+			s["parent_path"].Optional = false
+			s["parent_path"].Required = true
+			s["parent_path"].ForceNew = true
+			s["path"].Computed = true
+			s["serialized_dashboard"].Optional = true
```

Comment:
This is already optional in the Go SDK. You only need to customize things which are not expressed in the Go SDK.